openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -694,13 +694,16 @@ Cache-Control: no-store
 
 # Credential Endpoint {#credential-endpoint}
 
-The Credential Endpoint issues a Credential as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
+The Credential Endpoint issues one or more instances of a Credential Configuration or Credential Dataset as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
 
-Communication with the Credential Endpoint MUST utilize TLS. 
+Communication with the Credential Endpoint MUST utilize TLS.
 
-The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential.
+The Client may send a Credential Request to obtain:
 
-If the Access Token is valid for requesting issuance of multiple Credentials, it is at the Client's discretion to decide the order in which to request issuance of multiple Credentials requested in the Authorization Request.
+* one Credential of a particular Credential Configuration and Credential Dataset;
```

Comment:
```suggestion
* one Credential;
```
definition of a credential should be clear enough, no need to mention credential configuration or credential dataset i think

Reply:
to me, this is not about precision, but using the terminology we agreed upon. if it is one Credential, by definition it has one credential configuratio and one credential dataset, so having that is redundant..